yolo_ros_node:
  ros__parameters:
    yolo_model: "yolo11n.pt"
    #yolo_model: "yolo8n.pt"
    input_rgb_topic: "/image_raw"
    input_depth_topic: "/camera/depth/points"
    subscribe_depth: False
    publish_annotated_image: True
    rgb_topic: "/yolo_ros/rgb_image"
    depth_topic: "/yolo_ros/depth_image"
    annotated_topic: "/yolo_ros/annotated_image"
    detailed_topic: "/yolo_ros/detection_result"
    threshold: 0.5
    device: "0"
